spec/policies/page_policy_spec.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Documentation::PagePolicy do
+  fixtures :users, 'documentation/permissions', 'documentation/pages', 'documentation/workspaces'
+
+  let(:super_admin) { users(:super_admin) }
+  let(:read_user) { users(:read_user) }
+  let(:write_user) { users(:write_user) }
+  let(:full_user) { users(:full_user) }
+  let(:non_member_user) { users(:non_member_user) }
+
+  let(:page) { documentation_pages(:comedor_home_page) }
+
+  describe '#show?' do
+    it 'returns true for all users' do
+      expect(described_class.new(read_user, page).show?).to be_truthy
+      expect(described_class.new(write_user, page).show?).to be_truthy
+      expect(described_class.new(full_user, page).show?).to be_truthy
+    end
```

Comment:
We should test the policy like so:
```
  permissions :show? do
    it 'grants access to user with read permission' do
      expect(described_class).to permit(read_user)
    end
  end
```

Reply:
I dont think we need to use this... We are not installing pundit on the Gemfile and as is public repo... 
So we need to validate if return true or false, not validate thru Pundit....